stable/kube-ingress-aws-controller/templates/daemonset.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1beta2
+kind: DaemonSet
+metadata:
+  name: {{ template "kube-ingress-aws-controller.fullname" . }}
+  labels:
+    app: {{ template "kube-ingress-aws-controller.name" . }}
+    chart: {{ template "kube-ingress-aws-controller.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+{{- if .Values.daemonset.annotations }}
+  annotations:
+{{ toYaml .Values.daemonset.annotations | indent 4 }}
+{{- end }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "kube-ingress-aws-controller.name" . }}
+      release: {{ .Release.Name }}
+      component: ingress
```

Comment:
The DaemonSet itself should also have the `component` label.

Reply:
Fixed. Can you please check now?